03/app.js

```diff
@@ -1,3 +1,28 @@
-export default function randomNumber(min, max) {
+export default randomNumber = (min, max) => {
+    if (areArgumentsValid(min, max)) {
+        const result = Math.random() * (max - min) + min;
 
+        return result;
+    }
+}
+
+function areArgumentsValid(min, max) {
+    if (!areNumbers(min, max)) {
```

Comment:
Cały czas podtrzymuje, że funkcje, które starają się wykonywać więcej niż jedną czynność są złe.
Ta funkcja nie robi 1 rzeczy jakby można było pomyśleć bo nie tylko sprawdza poprawność danych ale robi to dla 2 danych (to są dwie rzeczy :P). Problem jest taki, że teraz komunikat jest nie czytelny tj. nie wiadomo czy pierwsza czy druga liczba jest niepoprawna. Pamiętam funkcje z poprzedniego zadania i o ile tam jeszcze była do przełknięcia to tutaj już nie :P

powinno być `isArgumentValid()` i osobno sprawdzanie każdego z odpowiednim komunikatem.

Zwróć uwagę też, że przez to, że nie wiemy, który parametr jest nie prawidłowy to test też nie jest jednoznaczny.
Nie mamy pewności, że faktycznie przy zapisie `randomNumber(2, 'nnn')` kod działa dobrze bo w przypadku `randomNumber('nnn', 3)` będzie taki sam komunikat. - w zasadzie to przy  `randomNumber('mm', 'nnn')` również. Czyli mamy 3 przypadki i ten sam komunikat - nie wygląda to dobrze :P

PS. Wybacz, że się tak rozpisałem :P

Reply:
Ach, pisząc niektóre funkcje rzeczywiście zdarza mi się nie wyczuć tego, że wykonuje ona więcej niż jedną rzecz :/. Muszę to jakoś wyczuć, bo bardzo chcę osiągnąć poziom jak najbardziej czytelnych i optymalnych funkcji. 
Może jak będziesz mi tak zwracał na to uwagę to w końcu coś zaskoczy 😅. 
Dziękuję!